docs/layouts/partials/page-meta-links.html

```diff
@@ -1,19 +1,28 @@
 {{ if .Path }}
-{{ $pathFormatted := replace .Path "\\" "/" }}
 {{ $gh_repo := ($.Param "github_repo") }}
 {{ $gh_subdir := ($.Param "github_subdir") }}
 {{ $gh_project_repo := ($.Param "github_project_repo") }}
 {{ $gh_branch := (default "master" ($.Param "github_branch")) }}
 {{ if $gh_repo }}
 <div class="td-page-meta ml-2 pb-1 pt-2 mb-0">
-{{ $editURL := printf "%s/edit/%s/content/%s" $gh_repo $gh_branch $pathFormatted }}
-{{ if and ($gh_subdir) (.Site.Language.Lang) }}
-{{ $editURL = printf "%s/edit/%s/%s/content/%s/%s" $gh_repo $gh_branch $gh_subdir ($.Site.Language.Lang) $pathFormatted }}
-{{ else if .Site.Language.Lang }}
-{{ $editURL = printf "%s/edit/%s/content/%s/%s" $gh_repo $gh_branch ($.Site.Language.Lang) $pathFormatted }}
-{{ else if $gh_subdir }}
-{{ $editURL = printf "%s/edit/%s/%s/content/%s" $gh_repo $gh_branch $gh_subdir $pathFormatted }}
+{{ $editURL := printf "%s/edit/%s" $gh_repo $gh_branch }}
+{{ if .File }}
+  {{ if .File.LogicalName }}
+    {{ if $gh_subdir }}
+      {{ $editURL = printf "%s/%s" $editURL $gh_subdir }}
+    {{ end }}
+    {{ $editURL = printf "%s/content" $editURL }}
+    {{ if .Site.Language.Lang }}
+      {{ $editURL = printf "%s/%s" $editURL .Site.Language.Lang }}
+    {{ end }}
+    {{ if eq .File.LogicalName "_index.md" }}
+      {{ $editURL = printf "%s/%s" $editURL (path.Dir .File.Path) }}
+    {{ else }}
+      {{ $editURL = printf "%s/%s" $editURL .File.Path }}
+    {{ end }}
```

Comment:
```suggestion
    {{ $editURL = printf "%s/%s" $editURL .File.Path }}
```
This worked well for "_index.md" files, 
would you please check again?

Reply:
@t-kikuc I know, I kept spliting the _index.md and the rest because I thought maybe you guys wanna point the link to the folder instead of the _index.md file. I'll remove this condition

Reply:
@ntheanh201 I got it, thank you so much.
We'll restore the condition when we need that.